README.md

```diff
@@ -96,6 +96,41 @@ class Point:
   Schema: ClassVar[Type[Schema]] = Schema
 ```
 
+It is also possible to derive all schemas from your own base class
```

Comment:
Can we create a separate section in the README for that ?

Reply:
You could add a sentence or two about when this is useful, and add a link to marshmallow's documentation about custom base classes.